draft-ietf-tls-tls13.md

```diff
@@ -2301,14 +2303,22 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+Legacy algorithms
+: Indicates algorithms which are being deprecated because they use
+  algorithms with known weaknesses, specifically SHA-1 when used
+  either with RSA using RSASSA-PKCS1-v1_5 or ECDSA.  These values
```

Comment:
How about "specifically SHA-1, which is used in this context with either RSASSA-PKCS1-v1_5 or ECDSA"?